src/main/assets/scss/main.scss

```diff
@@ -80,6 +81,15 @@ $text-disabled: #505a5f;
   text-align: center;
 }
 
+.left-align-text {
+  text-align: left;
+}
+
+.link-without-formatting {
```

Comment:
use gov stylings here too please

Reply:
which gov style I need to use? I can see that in main.scss, there are already .right-align-text and .centre-align-text. I replicated the same pattern.

Reply:
please see https://github.com/alphagov/govuk-frontend/tree/main/src/govuk/components

Reply:
not use text-align custom class anymore.